config/kubernetes/overlays/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
```

Comment:
```suggestion
kind: Kustomization
resources:
```

Reply:
I think bases clearly delineates what the bases are vs what the resources are.

Reply:
`bases` is deprecated in favor of `resources` and throws a warning when kustomize is executed. https://github.com/kubernetes-sigs/kustomize/issues/2243

Reply:
Oh that's a good point.